### PR TITLE
Add a blog post discussing recent development.

### PR DESCRIPTION
**Pull request by pchote**
This could do with proofreading from another pair of eyes and probably some more light editing before merging.

### CODE REVIEW

**Comment by reaperrr** [comment]
In the modding paragraph, maybe put the "The first ..." and the "The second ..." parts on new lines, for better readability.

File: content/news/devupdate-20180610.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+This dichotomy between "Original Red Alert" and "Original OpenRA" has caused significant conflict on our forum and the community Discord channels during the last year. The discussion were reignited by the change to building auto-targeting in release-20171014, and have increased in severity due to recent discussions about finding a way to move Hinds back to Allies and removing Kill Bounties as a default feature. On one side of the issue are thoughts that the RA mod should abandon some of the changes that don't make sense in the world of Red Alert 1 (e.g. remove Kill Bounties, but not necessarily the Flak truck) and instead double down on the features that made the Command and Conquer series memorable. On the other side are thoughts that it was exactly these changes that made *OpenRA* great, and that it is an insult to our community to discard these features because of misplaced nostalgia.
```
Comment (reaperrr) [inline content/news/devupdate-20180610.md]: 2nd sentence, discussion*s*

**Comment by pchote** [comment]
Updated.  New image:

File: content/news/devupdate-20180610.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+<div style="text-align:center" markdown="1">
+<img src="/images/news/20180610-download.png" width="600" alt="OpenRA Download page">
+
+A sneak-preview of the new download page featuring Linux AppImages.
+
+</div>
+
+Our modding community has continued to be a major focus, driven in a large part by active discussions with and Pull Requests from the modders themselves. We have recently achieved two major modding milestones which we are looking forward to support from the next playtest:
+<ul>
+<li><strong>Feature parity between the Mod SDK and the main OpenRA mods</strong>: the Mod SDK can now package Linux AppImages alongside the existing macOS .apps and Windows installers. Mods now also integrated properly with the online and in-game server lists (no more "Unknown Mod (id)"), and modders can define their own acknowledgements text to be displayed in the in-game Credits dialog.</li>
```
Comment (reaperrr) [inline content/news/devupdate-20180610.md]: 2nd sentence: Either "Mods *are* now also integrated ..." or "Mods now also integrate ..." (no d at end of integrate).

File: content/news/devblog-20180610.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+This dichotomy between "Original Red Alert" and "Original OpenRA" has caused significant conflict among our players and contributors on the forum and the community Discord channels. These discussions were reignited last year by the change to building auto-targeting, and have increased in passion with recent discussions about finding a way to move Hinds back to Allies and removing Kill Bounties as a default feature. On one side of the issue are thoughts that the RA mod should abandon some of the changes that don't make sense in the world of Red Alert 1 (e.g. Kill Bounties, but not the Flak Truck) and instead double down on the things that made the Command and Conquer series memorable. On the other side are thoughts that it is exactly these changes that made *OpenRA* great, and that it is an insult to our community to discard these features motivated by misplaced nostalgia.
```
Comment (GraionDilach) [inline content/news/devblog-20180610.md]: move Hinds back to Soviets, no?

**Comment by pchote** [comment]
Updated.

**Pull request merged**